workflow-issue-priority.md

```diff
@@ -35,8 +35,6 @@ Finish the task presently assigned to you, commit, then work on the issue.
 
 Finish your story, commit, but don’t start any new story before this issue is fixed.
```

Comment:
I'm okay with this as written (and for the future) but I understand if we need to do something to "ease" into this during this sprint or next. I'm sure that we're a few too many High tickets away from having this be immediately feasible.

I'm also okay with changing the work "fixed" into "addressed" and running with that for a while. If that turns out not to work we can change it back to "fixed" at a later date. Thoughts?